rfcs/react-components/convergence/long-term-support-plan.md

```diff
@@ -0,0 +1,133 @@
+# RFC: Long-term support (LTS) plan
+
+Geoff Cox, Tudor Papa, Gouttierre Gomes
+
+2024-02-06
+
+## Summary
+
+We need to establish a long-term support (LTS) plan to set expectations with partners and the open-source community. This includes building a plan for the sunset and retirement of older versions of Fluent.
+
+For example, this issue [Fluent v8 LTS/EOL (long-term support / end of life)?](https://github.com/microsoft/fluentui/discussions/29100) brought up the question.
+
+## Background
+
+We currently support v7, v8, and v9 versions of Fluent UI React and v0 of Northstar/Stardust.
+
+- v0 is active in the master branch. It is still supported by Teams, but all new feature work happens in v9.
+- v7 is in sunset. Only critical issues are fixed. The last fix was over 6 months ago (2023-08-17).
+- v8 is active in the master branch. Important issues raised by partners are fixed and partners actively develop on v8.
+- v9 is active in the master branch. It is Fluent's focus for feature development.
+
+## Problem statement
+
+We need to:
+
+1. Communicate LTS plans for each version to partners and the open-source community.
+2. Plan for sunset/retirement of old versions.
+3. Coordinate with partners that contribute to or own packages in our repository.
```

Comment:
Consider adding:

"4. Define and create an ongoing process and rhythm for how we will handle major changes to the repo so consumers can plan accordingly."

Reply:
I think that make the scope of this RFC too large to remain practical. This RFC can explain how the LTS plan would work with possible future versions, but it shouldn't turn into the entire operating manual for the team :- )